src/net/larse/lcms/algorithms/VCT.java

```diff
@@ -606,29 +487,13 @@ private void analyzeUDist() {
     //     min2Ud = this.ud[COMP];
     //   }
     // }
-    //
-//    double maxVi = Doubles.max(this.ud[NDVI]);
-//    double minUd = 9999.0;
-//    double min2Ud = 9999.0;
-//
-//    for (int i = 0; i < this.numYears; i++) {
-//      double cmp = this.ud[COMP][i];
-//      if (cmp < minUd) {
-//        minUd = cmp;
-//      }
-//      if (cmp > minUd && cmp < min2Ud) {
-//        min2Ud = cmp;
-//      }
-//    }
-//    if (min2Ud - minUd < FOR_THR_MAX) {
-//      min2Ud = minUd;
-//    }
 
-    //Yang incorporate Matt's new logic and consolidate calculation
-    //keep track of bottom two ud composite values
-    double maxVi = Double.NEGATIVE_INFINITY; //Doubles.max(this.ud[NDVI]);
-    double minUd = Double.NEGATIVE_INFINITY; //Doubles.min(this.ud[COMP]);
-    double min2Ud = Double.POSITIVE_INFINITY; //Doubles.max(this.ud[COMP]);
+    // double maxVi = Double.NEGATIVE_INFINITY; //Doubles.max(this.ud[NDVI]);
+    // double minUd = Double.NEGATIVE_INFINITY; //Doubles.min(this.ud[COMP]);
+    // double min2Ud = Double.POSITIVE_INFINITY; //Doubles.max(this.ud[COMP]);
+    double maxVi = Doubles.max(this.ud[NDVI]);
+    double minUd = 9999.0;
+    double min2Ud = 9999.0;
```

Comment:
I think we can use the new code (e.g. uncomment 491-493).  Kept in here for consistency testing